src/_posts/languages/nodejs/2000-01-01-angularjs.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Deploying AngularJS on Scalingo
+nav: AngularJS
+modified_at: 2023-02-16 00:00:00
+tags: nodejs angularjs
+index: 4
+---
+
+AngularJS applications can be hosted on Scalingo. It requires some specific instructions as usually the application generate static files during the deployment. And these files requires a web server to be served to your clients.
+
+Deploying an AngularJS application requires to use the [Multi Buildpacks]({% post_url platform/deployment/buildpacks/2000-01-01-multi %}). It will be used to generate the static files using Node.js during the deployment, and run a Nginx to serve the generated files.
+
+Create a `.buildpacks` file:
+
+```
+https://github.com/Scalingo/nodejs-buildpack
+https://github.com/Scalingo/nginx-buildpack
+```
+
+Configure Nginx to serve the generated files by creating a `nginx.conf` file. The following configuration assumes that the files have been generated in the `dist` folder:
+
+```nginx
+root /app/dist;
+
+location / {
+  try_files $uri $uri/ /index.html =404;
+}
+```
+
+Then instruct Scalingo how to start the application with a [`Procfile`]({% post_url platform/app/2000-01-01-procfile %}):
```

Comment:
"Instruct" doesn't seem that smooth to me 
```suggestion
Then tell Scalingo how to start the application with a [`Procfile`]({% post_url platform/app/2000-01-01-procfile %}):
```

Reply:
"tell": de ce que je sais, ce mot signifie plutôt "raconter" dans le sens "raconter des histoires".
"instruct" (https://www.wordreference.com/enfr/instruct) serait dans le sens "donner l'ordre"